springboot/src/main/java/io/temporal/samples/springboot/SamplesController.java

```diff
@@ -64,4 +72,52 @@ public String metrics(Model model) {
     model.addAttribute("sample", "SDK Metrics");
     return "metrics";
   }
+
+  @GetMapping("/update")
+  public String update(Model model) {
+    model.addAttribute("sample", "Synchronous Update");
+    model.addAttribute("products", productRepository.findAll());
+    return "update";
+  }
+
+  @GetMapping("/update/inventory")
+  public String updateInventory(Model model) {
+    model.addAttribute("products", productRepository.findAll());
+    return "update :: inventory";
+  }
+
+  @PostMapping(
+      value = "/update/purchase",
+      consumes = {MediaType.APPLICATION_JSON_VALUE},
+      produces = {MediaType.TEXT_HTML_VALUE})
+  ResponseEntity purchase(@RequestBody Purchase purchase) {
+    PurchaseWorkflow workflow =
+        client.newWorkflowStub(
+            PurchaseWorkflow.class,
+            WorkflowOptions.newBuilder()
+                .setTaskQueue("UpdateSampleTaskQueue")
+                .setWorkflowId("NewPurchase")
+                .build());
+    WorkflowClient.start(workflow::start);
+
+    // send update
+    try {
+      workflow.makePurchase(purchase);
```

Comment:
Why don't you make use of the return value `makePurchase`?

Reply:
updated. nice catch